### DISCUSSION TITLE
ZuriHac - Participation

### DISCUSSION THREAD

**Discussion by bolt12** [ludat/conferer#99] [General]
Hey I'm looking to contribute to `conferer` at ZuriHac and was wondering if is there any information about some higher level details about this project like its architecture, project structure, etc... If not maybe we could start there and this thread could be used as a reference for other people if they want to join.

We could also have a FAQ section that answers simple questions about the inner workings of `conferer` too, like: Will `conferer` only consider one source of configuration format, i.e. does `conferer` check every possible source (CLI, config file, etc?)? Does any format has some kind of priority? Is it possible to, for example, provide the PORT as a CLI flag and then use a JSON config file for the rest?

**Comment by ludat** [comment]
Hi! 

First, I'm very sorry for not answering before :disappointed:, I've been quite busy (so much I couldn't participate in this year's ZuriHac).

So right now the best site for docs is the website: [conferer.ludat.io](https://conferer.ludat.io/docs/core-concepts). it's still far from perfect but it's a start, the docs generated in hackage also have _some_ information regarding the implementation.

Also now that I've made the site, I can use the README for more developer related info (like contributing guidelines, FAQ and a roadmap)

Finally two things: are you still interested in contributing? if so, do you have some feature in mind that you'd like to see happening?

**Comment by bolt12** [comment]
@ludat Don't worry about it! Nice work on the website docs! I'll have a look to see if it answers some of my questions.

I am still happy to contribute but at a slower pace. I do not have any feature in mind, I just thought this project to be interesting :smile:
